pennylane_sf/remote.py

```diff
@@ -159,17 +161,15 @@ def var(self, observable, wires, par):
         return samples_variance(self.samples)
 
     def probability(self, wires=None):  # pylint: disable=missing-function-docstring
-        all_probs = all_fock_probs_pnr(self.samples)
-
-        # Extract the cutoff value by checking the number of Fock states we
-        # obtained probabilities for
-        cutoff = all_probs.shape[0]
+        fock_probs = all_fock_probs_pnr(self.samples)
+        cutoff = fock_probs.shape[0]
+        diff = max(self.cutoff - cutoff, 0)
+        all_probs = np.pad(fock_probs, [(0, diff)] * self.num_wires)
```

Comment:
Ah, so if the 'actual' cutoff is below the set cutoff, the fock probabilities are just padded with 0s?